Failed to compile on Fedora 40

Towards the end of the make command, I get a lot of errors like this one :

/usr/include/zlib.h:1842:1: error: ‘Z_EXTERN’ does not name a type; did you mean ‘ZEXTERN’?
 1842 | Z_EXTERN int              Z_EXPORT inflateUndermine (z_stream *, int);
      | ^~~~~~~~
      | ZEXTERN

--- a/cmake/recipes/geogram.cmake
+++ b/cmake/recipes/geogram.cmake
@@ -38,7 +38,7 @@ endif()
 ################################################################################
 
 include(CPM)
-CPMAddPackage("gh:polyfem/geogram#d38b11a32dd68e23db7a97dd7de31b43478bff2d")
+CPMAddPackage("gh:iiiian/geogram#3df437b17fae3b666436d3a2a476aab45f95cfcb")
 
 find_path(GEOGRAM_SOURCE_INCLUDE_DIR
 		geogram/basic/common.h

--- a/cmake/recipes/geogram.cmake
+++ b/cmake/recipes/geogram.cmake
@@ -38,7 +38,7 @@ endif()
 ################################################################################
 
 include(CPM)
-CPMAddPackage("gh:iiiian/geogram#3df437b17fae3b666436d3a2a476aab45f95cfcb")
+CPMAddPackage("gh:polyfem/geogram#0402d16ad5be8e892db239ac38ca724f9497dbd3")
 
 find_path(GEOGRAM_SOURCE_INCLUDE_DIR
 		geogram/basic/common.h

--- a/cmake/recipes/geogram.cmake
+++ b/cmake/recipes/geogram.cmake
@@ -38,7 +38,7 @@ endif()
 ################################################################################
 
 include(CPM)
-CPMAddPackage("gh:polyfem/geogram#3191ff581d4087ebddff735a21bf3ca2415f53be")
+CPMAddPackage("gh:polyfem/geogram#d38b11a32dd68e23db7a97dd7de31b43478bff2d")
 
 find_path(GEOGRAM_SOURCE_INCLUDE_DIR
 		geogram/basic/common.h

--- a/cmake/recipes/paraviewo.cmake
+++ b/cmake/recipes/paraviewo.cmake
@@ -8,4 +8,4 @@ endif()
 message(STATUS "Third-party: creating target 'paraviewo::paraviewo'")
 
 include(CPM)
-CPMAddPackage("gh:polyfem/paraviewo#7af5fc9a2c5c28c7ab7107964faf25ba20f1cdae")
+CPMAddPackage("gh:polyfem/paraviewo#1c01aae7ad37020f751eb598c62ae4ae74a3ada7")

--- a/cmake/recipes/paraviewo.cmake
+++ b/cmake/recipes/paraviewo.cmake
@@ -8,4 +8,4 @@ endif()
 message(STATUS "Third-party: creating target 'paraviewo::paraviewo'")
 
 include(CPM)
-CPMAddPackage("gh:polyfem/paraviewo#50e75b3d408fb1bb5d46cadc2126a5a71101c5e4")
+CPMAddPackage("gh:polyfem/paraviewo#7af5fc9a2c5c28c7ab7107964faf25ba20f1cdae")

--- a/cmake/recipes/paraviewo.cmake
+++ b/cmake/recipes/paraviewo.cmake
@@ -8,4 +8,4 @@ endif()
 message(STATUS "Third-party: creating target 'paraviewo::paraviewo'")
 
 include(CPM)
-CPMAddPackage("gh:polyfem/paraviewo#9ac491eb3a2a28b6725bf252ac3bf47c78882fb8")
+CPMAddPackage("gh:polyfem/paraviewo#50e75b3d408fb1bb5d46cadc2126a5a71101c5e4")

